draft-ietf-privacypass-architecture.md

```diff
@@ -126,23 +126,47 @@ the security of all participating entities.
 The following terms are used throughout this document:
 
 Client:
-: An entity that seeks authorization to an Origin.
+: An entity that seeks authorization to an Origin. Using {{?RFC9334}} terminology,
+Clients implement the RATS Attester role.
+
+Token:
+: A cryptographic authentication message used for authorization decisions.
 
 Origin:
-: An entity that redeems tokens presented by Clients.
+: An entity that consumes tokens presented by Clients and uses them to make authorization decisions.
+
+Challenge:
+: The mechanism by which Origins request tokens from Clients.
+
+Redemption:
+: The mechanism by which Clients present tokens to Origins for consumption.
 
 Issuer:
 : An entity that issues tokens to Clients for properties
   attested to by the Attester.
 
+Issuance:
+: The mechanism by which an Issuer produces tokens for Clients.
+
 Attester:
 : An entity that attests to properties of Clients for the
-  purposes of token issuance.
+  purposes of token issuance. Using {{?RFC9334}} terminology, Attesters implement
+  the RATS Verifier role.
 
 Attestation procedure:
 : The procedure by which an Attester determines whether or not a Client
   is trusted with a specific set of properties for token issuance.
 
+Anonymization service:
```

Comment:
Does this really need to be a definition, since it only used once below?

Reply:
If anything, let's move more of this text below where we mention the service.

Reply:
Done!